src/validation.cpp

```diff
@@ -1890,14 +1891,14 @@ bool static DisconnectTip(CValidationState& state)
     // Resurrect mempool transactions from the disconnected block.
     std::vector<uint256> vHashUpdate;
     for (const auto& txIn : block.vtx) {
-        const CTransaction& tx = *txIn;
+        const auto& tx = txIn;
```

Comment:
(7383441fed6b6ba274b9dab1f193e225d78badf1)
nit: can't we just rename  `txIn`? e.g.
```cpp
for (const auto& tx : block.vtx)  {
```

Reply:
yes, leftover from the previous code. Done.